### ISSUE TITLE
Provide the IPNS link directly anywhere on TEW's home page.

### ISSUE THREAD

**Issue by styromaniac** [PR0M3TH3AN/eternal-word#17]
It's a little complicated how you tell people to copy the CID. There should be an IPNS link provided directly, IMHO.

**Comment by PR0M3TH3AN** [comment]
I agree completely! When I first developed that documentation IPNS was very unreliable so I avoided using it, but it's probably time to revisit that to simplify the process. As soon as I get some time I'll look into it.
